src/com/comeet/utilities/Validator.java

```diff
@@ -0,0 +1,15 @@
+package com.comeet.utilities;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Validator {
+    
+    private static final Pattern VALID_EMAIL_ADDRESS_REGEX = 
```

Comment:
I had used a regex pattern based off of common email address patterns. I did some follow up and found a pattern that conforms to RFC 822 (http://www.ietf.org/rfc/rfc0822.txt?number=822) that I will include in my next pull request (http://www.ex-parrot.com/pdw/Mail-RFC822-Address.html)